src/libslic3r/Fill/FillBase.cpp

```diff
@@ -1838,18 +1838,18 @@ static inline void base_support_extend_infill_lines(Polylines &infill, BoundaryI
     const auto      dist_min_y      = coord_t(line_spacing * 0.5);
 
     for (ContourIntersectionPoint &cp : graph.map_infill_end_point_to_boundary) {
-        const Points                &contour         = graph.boundary[cp.contour_idx];
-        const std::vector<double>   &contour_param   = graph.boundary_params[cp.contour_idx];
-        const Point                 &pt              = contour[cp.point_idx];
-        const bool                   first           = graph.first(cp);
-        int                          extend_next_idx = -1;
-        int                          extend_prev_idx = -1;
-        coord_t                      dist_y_prev;
-        coord_t                      dist_y_next;
-        double                       arc_len_prev;
-        double                       arc_len_next;
-
-        if (! graph.next_vertical(cp)){
+        const Points                &contour         { graph.boundary[cp.contour_idx] };
```

Comment:
Do these generate compiler warnings? If not, we might want to keep the old way of initializing variables just to make code style consistent.